examples/specs_integration/src/sync.rs

```diff
@@ -0,0 +1,73 @@
+mod entity_create;
+mod entity_delete;
+mod entity_move;
+mod godot_sync_state;
+
+use godot::*;
+use specs::prelude::*;
+
+pub use self::entity_create::GdEntityCreation;
+pub use self::entity_delete::GDDeleteOldSpatials;
+pub use self::entity_move::GdEntityTransform;
+pub use self::godot_sync_state::GodotStateSync;
+use nalgebra::Isometry3;
+
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub enum ExampleScene {
+    ExampleCube,
+}
+
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub enum Template3dName {
+    // Create a new enum for every different thing that is in your game.
+    //  Module(ModuleDrawType), -- In my game I have a list of modules that you can build
+    //  SpaceObject(SpaceSubScenes), -- In my game I have stuff related to spaceships
+    ExampleScenes(ExampleScene),
+}
+
+// Where is something in space, really?
+pub struct Pos(pub Isometry3<f32>);
+
+impl Component for Pos {
+    type Storage = FlaggedStorage<Self, DenseVecStorage<Self>>;
+}
+
+#[derive(Default, Debug, Clone)]
+pub struct GdSceneLoc {
+    /// GdSceneLoc is how you specify what sort of Godot Spatial you want to create, what to call it etc.
+    pub parent_name: &'static str,

```

Comment:
This doc comment will apply to `parent_name`, instead of `GdSceneLoc` which it's obviously intended for. It should be moved above `GdSceneLoc` instead. Similar doc comments are also seen elsewhere.